docs/providers/aws/events/event-bridge.md

```diff
@@ -65,6 +65,30 @@ functions:
                 - cloudformation.amazonaws.com
 ```
 
+You can build a more robust pattern matching thanks to [content filtering](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-event-patterns-content-based-filtering.html).
+
+For example, if you defined a s3 bucket with EventBridge notification enabled, you can handle the same event as the default s3 from serverless:
+
+```yml
+functions:
+  myFunction:
+    handler: index.handler
+    events:
+      - eventBridge:
+          pattern:
+            source:
+              - aws.s3
+            detail-type:
+              - Object Created
+            detail:
+              bucket:
+                name:
+                  - photos
+              object:
+                key:
+                  - prefix: 'uploads/'
+```
```

Comment:
I agree about the documentation link.

About the `prefix`, it was just an example of the content filtering. IMHO it's interesting to have at least one example.

Reply:
Right, but I was really confused by the example, maybe this is just me. I couldn't understand if there was anything particular about S3 here, like why was it mentioned.

Actually, re-looking at the page (https://www.serverless.com/framework/docs/providers/aws/events/event-bridge#setting-up-event-pattern-matching) I realize it's not that long, so why not add an extra example. Especially since filtering is a huge feature of EventBridge events. I'd suggest then rephrasing the sentence to make it clearer?

E.g. something like:

> Here is an example that uses "[prefix matching](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-event-patterns-content-based-filtering.html#eb-filtering-prefix-matching)" to filter EventBridge events produced by S3:

Reply:
I rephrased the sentence

Reply:
Thanks!